specs/market-api.yaml

```diff
@@ -641,39 +604,86 @@ paths:
           to return to the Negotiation phase by sending a new Proposal.
 
         * `Cancelled` - Indicates that the Requestor himself has called
-         `cancelAgreement`, which effectively stops the Agreement handshake.
+          `cancelAgreement`, which effectively stops the Agreement handshake.
 
       responses:
-        '200':
+        200:
```

Comment:
Code `200` should mean `Approved` only. No body is needed in this case.
For other results, we should use `410` gone.

Reply:
OK